ArduPlane/quadplane.h

```diff
@@ -101,7 +101,7 @@ class QuadPlane
     }
 
     // return desired forward throttle percentage
-    int8_t forward_throttle_pct(void);        
+    int8_t forward_throttle_pct(bool tiltrotor);        
```

Comment:
you could default this to false and maybe make the bool return_disarmed or something. tilltrotor only makes sense from the context of the function calls but not within the function itself

Reply:
saved a few lines of code and (I think) eliminated confusion by moving the arming check up to the caller (this is only called from 2 places)